experimental/datacommons-js/example.html

```diff
@@ -26,6 +26,12 @@ <h2>Bar Chart</h2>
       comparisonVariables='["Median_Income_Person_15OrMoreYears_Male_WithIncome", "Median_Income_Person_15OrMoreYears_Female_WithIncome"]'
       comparisonPlaces='["geoId/01", "geoId/02", "geoId/04"]'
     ></datacommons-bar>
+    <datacommons-bar
+      title="Median income by gender"
```

Comment:
nit: use double quote as the outer quote to make it consistent across args

Reply:
Leaving this as is since we are planning on switching to space-separated values